packages/now-next/src/utils.ts

```diff
@@ -1110,7 +1110,9 @@ export function addLocaleOrDefault(
   if (!routesManifest?.i18n) return pathname;
   if (!locale) locale = routesManifest.i18n.defaultLocale;
 
-  return locale ? `/${locale}${pathname}` : pathname;
+  return locale
+    ? `/${locale}${pathname === '/index' ? '' : pathname}`
+    : pathname;
```

Comment:
Does the second `pathname` here also need to do the `/index` normalization?

Reply:
No, the index only needs to be handled this ways with i18n, without i18n the file is located at `index.html`